### PR TITLE
added info about default region and finding cloud details

### LINKED ISSUE #1167: Add show-clouds to clouds.md
After you use `juju list-clouds`, if you want more detail about a particular cloud, use:

    juju show-cloud azure

Seems like useful info to include in the main cloud page.

### PR DESCRIPTION
**Pull request by matthewhelmke**
@juju/docs
Fixes #1167 and #1168

### CODE REVIEW

File: src/en/clouds.md [hunk 1]
```diff
@@ -39,6 +39,22 @@ rackspace    rackspace   DFW, ORD, IAD, LON, SYD, HKG
 
 This lists the cloud name (which you will use to specify the cloud you want to 
 use), its type (the API used to control it) and the known regions for the cloud.
+The default region for a cloud is shown first in the list-clouds output, so
+in the above, `us-east-1` is the default region for an aws cloud. To specify
+a different region during controller creation, use:
+```bash
+juju bootstrap mycontroller aws/us-west-2
```
Comment (pmatulis) [inline src/en/clouds.md]: We can mention region defaults (I didn't know this) but I think the actual example command should be in [controllers-creating](https://jujucharms.com/docs/devel/controllers-creating) and possibly provide a link to that page.

**Comment by evilnick** [comment]
Yes, the region stuff should go with the bootstrap command. Just put it on that page instead and link it from here "(To specify a different region, see xxxx)"

**Comment by matthewhelmke** [comment]
@pmatulis @evilnick 

That makes sense. I have moved the info.

File: src/en/clouds.md [hunk 2]
```diff
@@ -39,6 +39,19 @@ rackspace    rackspace   DFW, ORD, IAD, LON, SYD, HKG
 
 This lists the cloud name (which you will use to specify the cloud you want to 
 use), its type (the API used to control it) and the known regions for the cloud.
+The default region for a cloud is shown first in the list-clouds output, so
+in the above, `us-east-1` is the default region for an aws cloud. To specify
+a different region, see [Creating a controller](./controllers-creating.html).
```
Comment (pmatulis) [inline src/en/clouds.md]: Now it looks confusing. I would put this link under the global `set-default-region` command below and say like: "To set a non-default region during the creation of a controller see *link*..."

Comment (matthewhelmke) [inline src/en/clouds.md]: I hear what you are saying. I don't feel strongly either way, although my original thought in placing it where I did is that it creates a nice segue from "here is how to find your default region" to "set the default region to something else". Putting "use a region other than the default" either between those two or after makes sense to me. Hmm. @evilnick , do you have a preference?

Comment (matthewhelmke) [inline src/en/clouds.md]: Ahh, I see he is out today. In that case, I'll merge as-is and we can revisit the location for the info again later.

**Pull request merged**